### PR TITLE
Seekers of the Storm compatibility

### PR DESCRIPTION
**Pull request by Phreelosu**
just made the recompiling changes

from net standard 2.0 -> 2.1

bumped unity and wwise ver, made the r2api split and all that

### CODE REVIEW

**Comment by Phreelosu** [comment]
wait dont merge yet, I found an issue

**Comment by tieeeeen1994** [comment]
Pardon me, but what is the goal, and what mod still uses this? :O

**Comment by Phreelosu** [comment]
Gradius Option Seed uses it, and I kinda want to revive it for the new RoR2 DLC
can be merged now

**Comment by cheeeeeeeeeen** [comment]
Wow. The mod is still in working order?

File: ChensHelpers/GeneralHelpers/DropletGenerator.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public static void Update()
                 PickupDropletController.CreatePickupDroplet(
                     spawnList[Run.instance.spawnRng.RangeInt(0, spawnList.Count)],
                     t.position, new Vector3(0f, -5f, 0f)
-                );
+                ); //I wont change this because im lazy, and pseudopulse already made a patcher for this specific method, so I'll just make that a dependency if its an absolute must
```
Comment (tieeeeen1994) [inline ChensHelpers/GeneralHelpers/DropletGenerator.cs:46]: What do you mean about this? Fill us in.

Comment (tieeeeen1994) [reply ChensHelpers/GeneralHelpers/DropletGenerator.cs:46]: You would also need to inform us how we can update the mod. I myself have forgotten how to update it.

If you need access, we can provide it.

Comment (Phreelosu) [reply ChensHelpers/GeneralHelpers/DropletGenerator.cs:46]: > What do you mean about this? Fill us in.

so Gearbox unfortunately removed CreatePickupDroplet in the Devotion update this year, and pseudopulse made a quick patcher for mods that used it here https://thunderstore.io/package/pseudopulse/GoxOfBears/

thats all there is to it

As for the update, merging this into the original repo, and then building the solution is the way to go
zip the contents (manifest.json, icon.jpg, README.md and the dll) and then upload to thunderstore under Chen's "team"

**Comment by cheeeeeeeeeen** [comment]
Approved.

**Comment by cheeeeeeeeeen** [comment]
Please be patient with me. I'm very busy nowadays.

**Pull request merged**